Sdext.tex

```diff
@@ -21,15 +21,23 @@ \section{Debug Mode} \label{debugmode}
 
 \begin{steps}{When executing code due to an abstract command, the hart stays in
     Debug Mode and the following apply:}
+\item All implemented instructions operate just as they do in M-Mode, unless
```

Comment:
```suggestion
\item All implemented instructions operate just as they do in M-mode, unless
```
To be consistent with the other bullet below (and the capitalization in the priv spec).